source/Nuke.Common/CI/GitHubActions/Configuration/GitHubActionsConfiguration.cs

```diff
@@ -1,9 +1,10 @@
-// Copyright 2019 Maintainers of NUKE.
+﻿// Copyright 2019 Maintainers of NUKE.
```

Comment:
If you remove this change, I can merge the PR :)

Reply:
Weird, that diff does not show on my machine.

Reply:
Ok, GitKraken was set to CLRF 🙄